### ISSUE TITLE
plotnet labels cut-off + DiagrammeR + Keras

### ISSUE THREAD

**Issue by vnijs** [fawda123/NeuralNetTools#20]
`plotnet` is a great function but sometimes the labels are cut-off (see example below). Reducing `cex_val` helps but it would be nice of the margins / padding could be adjusted as well. I tried using R's par(mar = c(0, 6, 0, 6)) but that doesn't seem to have the desired effect? Any suggestions? 

![image](https://user-images.githubusercontent.com/2998956/36057965-2eb544d8-0dcb-11e8-9e46-71b4f7c4d12d.png)

Question 1: Have you ever considered using [Diagammer](http://rich-iannone.github.io/DiagrammeR/) for plotnet plots? In addition to making great looking pots, this would also allow for things like tooltips that could provide additional information about the model or nodes (e.g., activation function used).

Question 2: Any idea if it would feasible to generate network plots for models trained using [Keras](https://keras.rstudio.com/)?

**Comment by vnijs** [comment]
Thanks @fawda123. Very useful!
